frontend/src/pages/GettingStarted.tsx

```diff
@@ -28,26 +28,32 @@ const options = {
 };
 
 const PAGE_CONTENT_MD = `
+<br/>
+
 ## Build your own pipeline
 
-Build an end-to-end ML pipeline with TFX  [Start Here](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252F0.1.40%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb) (Alpha)
+Build an end-to-end ML pipeline with TFX  [Start Here](https://console.cloud.google.com/ai-platform/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fstorage.googleapis.com%252Fjiyongjung-public%252Ftfx%252Fnotebook%252Fcreate_a_pipeline_using_a_template.ipynb) (Alpha)
 
-## Demos and Tutorials
+<br/>
 
+## Demos and Tutorials
 This section contains demo and tutorial pipelines.
 
 **Demos** - Try an end-to-end demonstration pipeline.
 
-  * [TFX pipeline demo](#/pipelines) - A trainer that does end-to-end distributed training for XGBoost models. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/parameterized_tfx_oss)
-  * [XGBoost Pipeline](#/pipelines) - Example pipeline that does classification with model analysis based on a public taxi cab BigQuery dataset. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm)
+  * [TFX pipeline demo](#/pipelines) - Classification pipeline with model analysis based on a public taxi cab BigQuery dataset. To build your own [start here!](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252F0.1.40%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb)
+  * [XGBoost Pipeline demo](#/pipelines) - An end-to-end distributed training for XGBoost models. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm)
 
+<br/>
 
 **Tutorials** - Learn pipeline concepts by following a tutorial.
 
   * [Data passing in python components](#/pipelines) - Shows how to pass data between python components. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components)
   * [DSL - Control structures](#/pipelines) - Shows how to use conditional execution and exit handlers. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures)
 
-  You can find additional tutorials and samples [here]()
+  Want to learn more? find additional tutorials and samples [here](https://github.com/kubeflow/pipelines/tree/master/samples)
```

Comment:
- Remove the leading white-space. Markdown is currently interpreting this line as part of the list.
- Suggested revision(making the link more descriptive):

  **Want to learn more?** [Learn from sample and tutorial pipelines.](https://github.com/kubeflow/pipelines/tree/master/samples)

- Consider changing the URL in the link to use the samples and tutorials in the docs:
  https://www.kubeflow.org/docs/pipelines/tutorials/

Reply:
>Remove the leading white-space. Markdown is currently interpreting this line as part of the list.

Done

>Suggested revision(making the link more descriptive):

Done

>Consider changing the URL in the link to use the samples and tutorials in the docs:
https://www.kubeflow.org/docs/pipelines/tutorials/

Done